src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManager.php

```diff
@@ -434,11 +434,15 @@ private function modifyCollection($data, $user, $breadcrumb): Collection
 
         $this->em->persist($collectionEntity);
 
+        $isSystemCollection = 2 === $collection->getType()->getId();
+
         if ($isNewLocale) {
             $this->domainEventCollector->collect(
                 new CollectionTranslationAddedEvent($collectionEntity, $collection->getLocale(), $data)
             );
-        } else {
+        } elseif (!$isSystemCollection) {
+            // do not dispatch modify event for system collections because the current implementation of the SystemCollectionManager 
```

Comment:
```suggestion
            // do not dispatch modify event for system collections because the current implementation of the SystemCollectionManager
```